store/store.go

```diff
@@ -32,6 +32,15 @@ const (
 
 var ErrNotFound = errors.New("eds not found in store")
 
+// AccessorGetter abstracts storage system that indexes and manages multiple eds.AccessorGetter by
+// network height.
+type AccessorGetter interface {
```

Comment:
I don't love this iface for one implementation but i'm also not gonna die on hill

Reply:
I know but I've added it to be able to properly test some edge cases